cves/CVE-2018-1999015.yml

```diff
@@ -64,7 +64,16 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A vulnerability that allows the system to read out of the array when reading the 
+  contents of an ASR file from user input. An ASF file is a video or audio file 
```

Comment:
Most readers won't what ASR or ASF files are, so make sure you (briefly) describe them both.

Reply:
ASR was actually a typo, I meant ASF. I will fix this